streamr-dev/streamr-platform
# [PLATFORM-1148] Fix unselectable dropdown in firefox

Chart range & port value select dropdowns were unselectable and closed immediately on click in Firefox.
This fixes that.

#### To Test

1. Open a canvas with a select dropdown port e.g. "unit" port in `Clock` or "window type" in the `Count` module.
2. Add a `Chart` module and note the "Range" dropdown in the chart body. 
3. Dropdowns should appear, and values should change if selected in Chrome, Safari & Firefox.
